paper.md

```diff
@@ -33,37 +33,41 @@ bibliography: paper.bib
-Tools such as `dcm2niix` [@Li_2016] can be used to convert individual DICOM files into named NIfTI and even can extract some additional extra metadata into sidecar `.json` files. 
-But NIfTI file format carries only basic metadata and `dcm2niix` does not instruct how to organize multiple files within a study.
-
-HeuDiConv was developed to provide flexible tooling for labs to be able efficiently and consistently convert collections of DICOM files into collections of NIfTI (and compressed archives of DICOMs) files in desired file system hierarchies.
-Since the inception of HeuDiConv in 2014, a community-driven standard Brain Imaging Data Structure (BIDS) [@GAC+16] was established, which formalized such datasets layout and storage of metadata.
-Since then the most frequent use-case for HeuDiConv became conversion of DICOM files into BIDS datasets.
-Standardization into BIDS facilitates not only reuse of already shared datasets but also streamlines data validation, curation, analysis, etc.
+In order to support efficient and transparent processing, as provided by cutting-edge Free and/or Open Source (FOSS) tools, neuroimaging data must be formatted according to equally open and accessible standards.
+The Brain Imaging Data Structure (BIDS) [@GAC+16] is an open standard designed for computational accessibility, operator legibility, and a wide and easily extendable scope of modalities — and is consequently used by numerous analysis and processing tools as the preferred input format.
+HeuDiConv (Heuristic DICOM Converter) enables flexible and efficient conversion of spatially reconstructed neuroimaging data from the DICOM format (quasi-ubiquitous in biomedical image acquisition systems, particularly in clinical settings) to BIDS, as well as other file layouts.
+This can be done either via a multi-stage operator input workflow (discovery, manual tuning, conversion) or via a fully automated process that can be seamlessly integrated into a data pipeline.
+HeuDiConv is written in Python, and supports the DICOM specification for input parsing, and the BIDS specification for output construction.
+The support for these standards is extensive, and HeuDiConv can handle complex organization scenarios such as arise for specific data types (e.g. multi-echo sequences, or single-band reference volumes).
+In addition to generating valid BIDS outputs, additional support is offered for custom output layouts.
+This is obtained via a set of example heuristiscs, which can be modified into supplemental heuristics expressed as simple Python functions, thus providing full flexibility and maintaining user accessibility.
+HeuDiConv further integrates with DataLad [@datalad], and can automatically prepare DataLad archives with optional obfuscation of sensitive data and metadata, including obfuscating patient visit timestamps in version control.
```

Comment:
I do not want to use `archive` when talking about DataLad for a number of reasons.  repository or dataset would fit better.

```suggestion
HeuDiConv further integrates with DataLad [@datalad], and can automatically prepare DataLad datasets with optional obfuscation of sensitive data and metadata, including obfuscating patient visit timestamps in version control.
```